### PR TITLE
[feature] Add styling for SeeMore, SeeLess links

### PR DESCRIPTION
**Pull request by Roka20012**
Hey dude!
I like your package and I am using it in the project
So **this feature describes the case that I need in a production project** for ex. I have different font sizes and color for the main text and this links so please check it and merge if this feature worth it :)
And **please update the minor version of the package this feature shouldn't affect on previous code**
Thanks in advance!

### CODE REVIEW

File: src/SeeMore/SeeMore.js [hunk 1]
```diff
@@ -136,14 +141,10 @@ class SeeMore extends React.Component {
 
     return (
       <Text
-        testID="SeeMore"
```
Comment (kashishgrover) [inline src/SeeMore/SeeMore.js:139]: Seems like this is an unexpected change

File: src/SeeMore/SeeMore.js [hunk 2]
```diff
@@ -136,14 +141,10 @@ class SeeMore extends React.Component {
...
         onLayout={isShowingMore ? undefined : this.onLayout}
         numberOfLines={isShowingMore ? undefined : numberOfLines}
-        {...this.panResponder.panHandlers}
```
Comment (kashishgrover) [inline src/SeeMore/SeeMore.js:142]: Same here, seems like this is an unexpected change

File: src/SeeMore/SeeMore.js [hunk 3]
```diff
@@ -122,10 +124,13 @@ class SeeMore extends React.Component {
...
         {...this.panResponder.panHandlers}
-        style={[style, { color: isLinkPressed ? linkPressedColor : linkColor }]}
-      >
+        style={[style, { color: isLinkPressed ? linkPressedColor : linkColor }]}>
         {isShowingMore ? null : <Text {...this.props}>...</Text>}
-        {isShowingMore ? ` ${seeLessText}` : ` ${seeMoreText}`}
+        {isShowingMore ? (
+          <Text style={seeLessTextStyle}>{seeLessText}</Text>
+        ) : (
+          <Text style={seeMoreTextStyle}>{seeMoreText}</Text>
```
Comment (kashishgrover) [inline src/SeeMore/SeeMore.js:132]: Seems like you have missed out on the space. If you notice the code before, I have mentioned the text as ` ${text}`

**Review by kashishgrover — CHANGES_REQUESTED** [review]
Thanks for submitting this pull request. Please make the changes that I have mentioned. I need to verify this on my end once as well.

**Comment by Roka20012** [comment]
@kashishgrover thanks for the comments man will fix it!
And one more feature I think we could make link like `text | React.Element`, in that case users of this package could even use their custom links like React Components

**Comment by kashishgrover** [comment]
@Roka20012 it's a good suggestion. Feel free to raise a task to track the same. You can label it with "good first issue"

File: src/SeeMore/SeeMore.js (hunk 3 above)
Comment (Roka20012) [reply src/SeeMore/SeeMore.js]: Good catch, done ✅

File: src/SeeMore/SeeMore.js (hunk 2 above)
Comment (Roka20012) [reply src/SeeMore/SeeMore.js:142]: Fixed ✅

ps. It's prettier :)

File: src/SeeMore/SeeMore.js (hunk 1 above)
Comment (Roka20012) [reply src/SeeMore/SeeMore.js]: Done ✅

**Comment by Roka20012** [comment]
> @Roka20012 it's a good suggestion. Feel free to raise a task to track the same. You can label it with "good first issue"

Seems like I don't have the rights to add a label

**Review by RigottiG — APPROVED** [review]

**Comment by kashishgrover** [comment]
@Roka20012 well it seems like you added it. I am merging this. Will pull it, test it, update the examples, and release another version in some time. Thanks for your contribution! :D

**Comment by kashishgrover** [comment]
Hi @Roka20012 

I spent some time to check your changes. I should have given you a proper review before merging. I ended up having to fix a few things myself. 

Here's my in depth review:

1. The link style passing is great, but I don't see why a user would need to pass separate styles for see more/see less links. Ideally, they should have the same styles. And so I ended up changing your `seeMoreTextStyle` and `seeLessTextStyle` into a single `linkStyle` property. I feel that it is way cleaner, because anyway naming of things in this project is not the most optimal. I want to make sure we don't add to confusion when adding more props. Let me know what you think.

2. Passing `React.Element` won't work out of the box, and I had to unfortunately revert some of your changes there. This is because the core also logic also depends on the `text.length` of the "see more" string. I don't have a need for this feature and don't feel that this could be something high in demand. It will need quite some work and we will have to calculate for this custom react element into the logic itself. If you feel that this is something you really need in this package, do dig in and raise another PR. And if you do, please make sure that you test it on the example in this project, update the Readme doc, add a screenshot, and maybe even add a unit test for it. I'd love to review it.

Thanks again for your contribution.

**Pull request merged**